tests/nodes/draw/test_bbox.py

```diff
@@ -66,8 +76,19 @@ def test_bbox_no_label(self, draw_bbox_no_labels, draw_bbox_show_labels):
         output_img_no_label = original_img.copy()
         output_img_show_label = original_img.copy()
         labels = ["Person"]
-        input1 = {"bboxes": bboxes, "img": output_img_no_label, "bbox_labels": labels}
-        input2 = {"bboxes": bboxes, "img": output_img_show_label, "bbox_labels": labels}
+        scores = [0.99]
+        input1 = {
+            "bboxes": bboxes,
+            "img": output_img_no_label,
+            "bbox_labels": labels,
+            "bbox_scores": scores,
+        }
+        input2 = {
+            "bboxes": bboxes,
+            "img": output_img_show_label,
+            "bbox_labels": labels,
+            "bbox_scores": scores,
+        }
         draw_bbox_no_labels.run(input1)
         # after running draw, should not be equal
         assert original_img.shape == output_img_no_label.shape
```

Comment:
To update unit test for the above changes.

Reply:
Update in the forked [PR](https://github.com/sabrimansor/PeekingDuck/pull/3), pending for review